tests/notebook-demo.ipynb

```diff
@@ -0,0 +1,1725 @@
+      "--- Bound     [min, max] : [ 6.000E+02, 3.200E+04] - Zero values observed as well\n",
+      "--- Matrix    [min, max] : [ 1.000E-01, 3.490E+04]\n",
+      "--- Executing CBC (Solvelink=2): elapsed 0:00:00.448\n",
+      "\n",
+      "COIN-OR CBC      49.3.0 7de46a92 Mar 18, 2025          LEG x86 64bit/Linux    \n",
+      "\n",
+      "COIN-OR Branch and Cut (CBC Library 2.10.11)\n",
+      "written by J. Forrest\n",
+      "*** Error Cannot open parameter file \"/scratch/htc/skrishna/xl2times/benchmarks/out/DemoS_003-all/cbc.opt\"\n",
+      "*** Error Error code = 2; No such file or directory\n",
+      "\n",
+      "Parallel mode: deterministic, using 39 threads\n",
+      "\n",
+      "Calling CBC main solution routine...\n",
+      "Perturbing problem by 0.001% of 5675.77 - largest nonzero change 0.0017997186 ( 0.001731433%) - largest zero change 0.0017604745\n",
+      "0  Obj 237924.97 Primal inf 691691.34 (20)\n",
+      "63  Obj 2123888.9 Primal inf 121822.03 (24)\n",
+      "108  Obj 3195606.7\n",
+      "Optimal - objective value 3195557.7\n",
+      "Optimal objective 3195557.669 - 108 iterations time 0.002, Presolve 0.00\n",
+      "\n",
+      "Solved to optimality.\n",
+      "--- Reading solution for model TIMES\n",
+      "***\n",
+      "*** Reading with solveopt=REPLACE (0)\n",
+      "***\n",
+      "--- Executing after solve: elapsed 0:00:00.696\n",
+      "--- scenario.run(35305) 8 Mb\n",
+      "--- scenario.run(35306) 8 Mb\n",
+      "--- TIMES Solve status: Optimal\n",
+      "--- scenario.run(37112) 8 Mb\n",
+      "--- Putfile END_GAMS /scratch/htc/skrishna/xl2times/benchmarks/out/DemoS_003-all/end_gams\n",
+      "--- Putfile QLOG /scratch/htc/skrishna/xl2times/benchmarks/out/DemoS_003-all/qa_check.log\n",
+      "--- GDX File /scratch/htc/skrishna/xl2times/benchmarks/out/DemoS_003-all/scenario.gdx\n",
+      "--- Profile Summary (2540 records processed)\n",
+      "     0.247   0.008GB     35282 Solver     TIMES\n",
+      "     0.170   0.008GB     35306 IF-ELSE    \n",
+      "     0.169   0.008GB     37112 GAMS Fini   \n",
+      "     0.142   0.008GB     35121 Put        \n",
+      "     0.002   0.008GB     15740 Assignment OPYEAR (15)\n",
+      "     0.002   0.008GB     16588 Assignment OPYEAR (820)\n",
+      "     0.002   0.008GB     18274 Equation   EQG_COMBAL (85)\n",
+      "     0.002   0.008GB     35282 Solve Fini TIMES (860)\n",
+      "     0.001   0.007GB      6564 IF-ELSE    \n",
+      "     0.001   0.007GB      7689 IF-ELSE    \n",
+      "*** Status: Normal completion\n",
+      "--- Job scenario.run Stop 06/02/25 09:45:09 elapsed 0:00:01.049\n",
+      "--- runmodel.gms(5) 2 Mb\n",
+      "*** Status: Normal completion\n",
+      "--- Job runmodel.gms Stop 06/02/25 09:45:09 elapsed 0:00:01.315\n",
+      "\n",
+      "dict_values([3195557.6688330006])\n",
+      "dict_values([])\n",
+      "None\n"
```

Comment:
These are the values of REG_OBJ, PAR_COMPRDL, and EQE_COMPRD in the resulting `scenario.gdx` file. Do they look as expected? Should I be reading the `scenario~Data_250602_094507.gdx` file instead?